openshift-infra/tower_vars.yml

```diff
@@ -62,3 +61,4 @@ tower_workflow_template: "0-Push_Button_Deploy_OpenShift_on_AWS"
 tower_workflow_template_extra_vars: "tower_workflow_template_extra_vars.yml"
 # Wire up job_templates in this file
 tower_workflow_template_schema_path: "tower_workflow_template_schema.yml"
+tower_workflow_job_launch: true
```

Comment:
I would expect this to be `false` by default and have the students execute the workflow